app/views/admin/questions/_option_fields.html.erb

```diff
@@ -1,10 +1,12 @@
 <div class="nested-fields">
   <div>
     <%= f.label :description, t(".answer") %>
+    <span class="require">*</span>
```

Comment:
sao ko dùng thuộc tính required anh

Reply:
https://stackoverflow.com/questions/11197671/use-css-to-automatically-add-required-field-asterisk-to-form-inputs